client/my-sites/github-deployments/components/installations-dropdown/use-live-installations.ts

```diff
@@ -125,6 +136,8 @@ export const useLiveInstallations = ( {
 		} );
 
 		if ( ! openedPopup ) {
+			dispatch( recordTracksEvent( 'calypso_hosting_github_app_install_failed' ) );
```

Comment:
We should check the URL before dispatching the event. It could be that it failed to open the auth popup too. Another option would be to dispatch a more generic event not tied to the action but rather to opening the popup itself.

Reply:
I will add which should always fire and we can use it at the first step on various funnels.

```js
	const onNewInstallationRequest = () => {
		dispatch( recordTracksEvent( 'calypso_hosting_github_app_open_auth_popup_requested' ) );
```